replacy/suggestion.py

```diff
@@ -179,7 +183,7 @@ def inflect(self, item, item_options, pattern, pattern_ref, doc, start, end):
                 # get all forms
                 item_options = (
                     seq(item_options)
-                    .map(lambda x: inflector.inflect_or_lookup(x, pos=None))
+                    .map(lambda x: self.inflector.inflect_or_lookup(x, pos=None))
```

Comment:
:exploding_head:

Reply:
my editor complained, and I thought... yeah, you are right editor, inflector is not defined!